admin/load.php

```diff
@@ -156,17 +156,33 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 				<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" aria-describedby="<?php echo esc_attr( "{$base_id}_description" ); ?>" disabled>
 				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" value="<?php echo $enabled ? '1' : '0'; ?>">
 				<?php
+				$message = perflab_can_load_module( $module_slug, true );
+				if ( $message ) {
+					echo esc_html( $message );
```

Comment:
See my other comment: Instead of changing the overall infrastructure function, since this is for now the only use-case, I would suggest we keep it simple and hard-code the `database/sqlite`-specific message here. This way we make the change in a way that allows us to think through a cleaner way to build support for this into a proper API, decoupled from this PR.

Reply:
Done